mne/utils/xarray.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+
+from .. import Epochs, Evoked
+from ..time_frequency import AverageTFR
+
+
+def to_xarray(mne_inst):
+    """Convert MNE object instance to xarray DataArray.
+
+    Parameters
+    ----------
+    mne_inst : Epochs | Evoked
+        The MNE object to convert.
+
+    Returns
+    -------
+    xarr : DataArray
+        The xarray object.
+    """
+    from xarray import DataArray
+    from mne.utils import _validate_type
+
+    _validate_type(mne_inst, (Epochs, Evoked, AverageTFR))
+
+    if isinstance(mne_inst, Epochs):
+        data = mne_inst.get_data()
+        dims = ('chan', 'trial', 'time')
```

Comment:
Can we call the dimension "channel"?

Reply:
I prefer shorter dimension names as you use them often during various operations, but I will change the name if this would be the commin preference. :)

Reply:
I don't think we're internally consistent on questions like this. E.g., in function names sometimes we use `channel` and sometimes `ch`. Also when exporting to data frames, the spectrum class makes a column `freq` (not `frequency`).  So I'm not really sure what to do WRT "channel".  However, I would say that `trial` is not ideal, it should be `epoch` (for e.g. resting state recordings cut into sequential chunks "trial" doesn't make sense)

Reply:
oh, yes I will definitelly change `trial` to `epoch`, especially that you can have multiple epochs from the same trial (for example successively presented items to remember).